spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,6 +56,7 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
```

Comment:
My preference was to use `allow(Spree::Config).to receive(:allow_backorders).and_return(false)` but unfortunately one of the tests fails in Travis. My another preference was to use `before` and `after` blocks instead of `around`. Hopefully it won't fail.